pyaedt/application/Design.py

```diff
@@ -1567,6 +1568,14 @@ def release_desktop(self, close_projects=True, close_desktop=True):
 
         """
         release_desktop(close_projects, close_desktop)
+        props = [a for a in dir(self) if not a.startswith('__')]
+        for a in props:
+            try:
+                del self.__dict__[a]
+            except Exception as e:
+                pass
```

Comment:
You can use the one line `self.__dict__.pop(a, None)` to remove keys if you are not sure if they exist. 
This will return `self.__dict__[a]` if key exists in the dictionary, and `None` otherwise. 

`del self.__dict__[a]` will raise a `KeyError` if the key is not in the dictionary, so I would use: `except KeyError:` instead of `except Exception as e:`. Unless of course there are other reasons why it would fail.

Reply:
ok. I'll change it accordingly and check that it still work with desktop_release